var/spack/repos/builtin/packages/lua/package.py

```diff
@@ -37,6 +39,9 @@ class Lua(Package):
     # luarocks needs unzip for some packages (e.g. lua-luaposix)
     depends_on('unzip', type='run')
 
+    # By default, we are missing lua.pc
+    patch('add-lua-pc-file.patch', when="+pc-file")
```

Comment:
Would this work:
```suggestion
    patch(
        "http://lua.2524044.n2.nabble.com/attachment/7666421/0/pkg-config.patch",
        sha256="208316c2564bdd5343fa522f3b230d84bd164058957059838df7df56876cb4ae"
        when="+pc-file"
    )
```
? If so we can avoid storing the patch in our repository.

Reply:
Oh neat! Didn't know I could do that.